src/hb-ot-color-colr-table.hh

```diff
@@ -150,9 +611,12 @@ struct COLR
   bool sanitize (hb_sanitize_context_t *c) const
   {
     TRACE_SANITIZE (this);
-    return_trace (likely (c->check_struct (this) &&
-			  (this+baseGlyphsZ).sanitize (c, numBaseGlyphs) &&
-			  (this+layersZ).sanitize (c, numLayers)));
+    return_trace (c->check_struct (this) &&
```

Comment:
This still doesn't check version for 0. I don't know what I'm not successfully communicating.

Reply:
I added explicit check for version 0. I think I misunderstood you. I thought "check for version 0" is to check the first few fields. :)

Reply:
Thanks.